str/str.go

```diff
@@ -19,3 +22,232 @@ func InSlice(input interface{}, expects ...interface{}) bool {
 	}
 	return false
 }
+
+// Return the remainder of a string after the first occurrence of a given value.
+func After(subject string, search string) string {
```

Comment:
This can be reduced to 
```go
func After(input string, search string) string {
	if len(search) == 0 {
		return input
	}
	results := strings.SplitN(input, search, 2)
	return results[len(results)-1]
}
```